app/models/workers_scaling/experiment_statistics_factory.rb

```diff
@@ -0,0 +1,19 @@
+module WorkersScaling
+  class ExperimentStatisticsFactory
+    @cache = ActiveSupport::Cache::MemoryStore.new expires_in: 10.minutes
```

Comment:
Użyjcie czegoś takiego:
```
    addresses = Rails.cache.fetch("#{service}_adresses", expires_in: 30.minutes) do
      InformationService.instance.get_list_of(service)
    end
```

Gdzie ``InformationService.get_list_of...`` to kod pobierający świeżą wartość.